src/components/Coverage.tsx

```diff
@@ -79,6 +80,7 @@ const Coverage = ({
   ownedHotspots,
   followedHotspots,
   showRewardScale,
+  showEarningsScale,
```

Comment:
It feels like `showRewardScale` and `showEarningsScale` could be combined into one prop. Maybe a discriminated string union would be better.